parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageReadStore.java

```diff
@@ -80,10 +80,12 @@ static final class ColumnChunkPageReader implements PageReader {
     private final byte[] dataPageAAD;
     private final byte[] dictionaryPageAAD;
 
+    ParquetMetricsCallback metricsCallback;
```

Comment:
It should be `private final`

Reply:
Changed

Reply:
Did you forget to push your local changes? @parthchandra

Reply:
Yes, I just pushed them now. 🤦🏾

Reply:
It would be better to mark it as final. I suggest getting `callback` from `ParquetReadOptions` and then we don't need to update the constructor and more reasonable.